docs/can-guides/commitment/recipes/file-navigator/file-navigator-simple.md

```diff
@@ -216,56 +212,72 @@ find a folder, we need to render its contents.
 
 ### Things to know
 
-- A template can call out to another registered _partial_ template with with [can-stache.tags.partial {{>PARTIAL_NAME}}] like the following:
+- A template can call out to another registered _partial_ template with [can-stache.tags.partial {{>PARTIAL_NAME}}] like the following:
 
   ```html
   {{>PARTIAL_NAME}}
   ```
 
-- You can register partial templates with [can-stache.registerPartial can.stache.registerPartial] like the following:
+- You can register an inline named partial within the current template [can-stache.tags.named-partial {{<PARTIAL_NAME}}] like the following:
 
   ```js
-  const template = can.stache.from("TEMPLATE_ID");
-  can.stache.registerPartial("PARTIAL_NAME", template);
+  Component.extend({
+    tag: "my-component",
+    view: `{{<partialView}} BLOCK {{/partialView}}`
+  });
   ```
 
-### The Solution
-
-Update the __HTML__ tab to:
-
-- Call to an `{{>entities}}` partial.
-
+- The registered inline named partial can be called 
+  [recursively](https://canjs.com/doc/can-stache.tags.named-partial.html#TooMuchRecursion) like the following:
 ```html
-<script type="text/stache" id="entities-template">
-  <span>{{name}}</span>
-  <ul>
-    {{#for(child of children)}}
-      <li class="{{child.type}} {{#if(child.hasChildren)}}hasChildren{{/if}}">
-        {{#eq(child.type, 'file')}}
-          📝 <span>{{child.name}}</span>
-        {{else}}
-          📁 <span>{{child.name}}</span>
-        {{/eq}}
-      </li>
-    {{/for}}
-  </ul>
-</script>
+{{<recursiveView}}
+  <div>{{name}} <b>Type:</b> {{#if(nodes.length)}}Branch{{else}}Leaf{{/if}}</div>
+  {{#each(nodes)}}
+    {{>recursiveView}}
+  {{/each}}
+{{/recursiveView}}
+
+{{>recursiveView yayRecursion}}`
 ```
-@highlight 9
+### The Solution
 
-Update the __JavaScript__ tab to:
+Update the __JAVSCRIPT__ tab to:
 
- - Register the `entities-template` as a partial:
+- Call to an `{{>entities}}` partial.
 
 ```js
-const template = can.stache.from("entities-template");
-can.stache.registerPartial("entities", template);
-
-const fragment = template(rootEntityData);
-
-document.body.appendChild( fragment );
+import { Component } from "//unpkg.com/can@5/core.mjs";
+
+Component.extend({
+  tag: "file-navigator",
+  view: `
+    {{<entities}}
+      <span>{{this.name}}</span>
+      <ul>
+        {{# for(child of this.children) }}
+          <li class="{{child.type}} {{# if(child.hasChildren) }}hasChildren{{/ if}}">
+            {{# eq(child.type, 'file')}}
+              📝 <span>{{child.name}}</span>
+            {{else}}
+              📁 {{entities child}}
```

Comment:
I would change this and `{{entities rootEntity}}` to the call syntax (`{{entities(rootEntity}}`).